R/alt_aggregate_counts.R

```diff
@@ -26,14 +26,13 @@ alt_aggregate_counts <- function(
     round_ <- ifelse(week_grouping, "week", "month")
 
     # read in ucla data and do the appropriate grouping
-    fac_long_df <- read_scrape_data(
-        date_cutoff = date_cutoff, all_dates = all_dates, wide_data = TRUE) %>%
```

Comment:
By using `read_scrape_data(wide = TRUE)` here, we're getting rid of the statewide rows in places like Texas and Ohio where we're intentionally collecting both facility-level AND statewide counts and want to use the statewide row. Instead, we want to read the long version (like we do in `calc_aggregate_counts` [here](https://github.com/uclalawcovid19behindbars/behindbarstools/blob/master/R/calc_aggregate_counts.R#L61)).